File: leetcode/nackwon/0001_TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] resultArr = new int[2];

        int intSize = nums.length;

        int sum = 0;
        for (int i=0; i<intSize; ++i) {
            for (int j=i+1; j<intSize; ++j) {

                if (i != j) {
                    sum = nums[i] + nums[j];

                    if( sum == target) {
                        resultArr[0] = i;
                        resultArr[1] = j;
                        return resultArr;
                    }
                }
            }
        }
        return null;
    }
}

File: leetcode/SVAbel/easy/0001_twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = []
        for i in range(0, len(nums)):
            for k, t in enumerate(nums):
                if nums[i] + nums[k] == target and i != k:
                    return [i, k]
